torch/csrc/jit/ir.cpp

```diff
@@ -941,7 +1017,7 @@ bool Node::isNondeterministic() const {
       "aten::randn_like(Tensor self, *, int dtype, int layout, Device device, bool pin_memory, MemoryFormat? memory_format=None) -> Tensor",
       "aten::randperm(int n, *, int? dtype, int? layout, Device? device, bool? pin_memory) -> Tensor"};
 
-  if (nondeterministic_ops.find(this) == nullptr) {
+  if (!isMemberOf(nondeterministic_ops)) {
```

Comment:
OT: how would we find out if a sig change in `native_functions.yaml` caused one of these to start missing? Also, will a new op with nondeterministic behavior need to know this exists?

Simple answer would be to make this a property of the native_ops decl (or whatever single source of truth we converge on)... ?

Reply:
These signatures are checked against registered ops. If we don't fine them, then there will be an error on program initialization. So typically when one edited native_ops, they quickly find that these need to get updated as well. The harder thing to detect is if someone adds a non-deterministic op.